code/modules/cm_marines/anti_air.dm

```diff
@@ -1,8 +1,8 @@
 var/obj/structure/anti_air_cannon/almayer_aa_cannon
```

Comment:
remove this variable

You don't need GLOB.AAgunLocation because you can just make a global variable called GLOB.aa_gun and then do GLOB.aa_gun.loc